La'Tonia/guess_game_practice.py

```diff
@@ -0,0 +1,30 @@
+# practice project courtesy favtutor:
+# https://favtutor.com/blog-details/7-Python-Projects-For-Beginners
+
+# adds random package to the code to generate unknown numbers
+import random
+# creates variable named number assigned to random package with random integer method containing parameters for start and end numbers
+number = random.randint(1, 10)
```

Comment:
some of these puzzles remind me of my program. I should pull some out for examples